Remotion/Data/DomainObjects.UnitTests/SetupFixture.cs

```diff
@@ -15,87 +15,32 @@
-
-        SqlConnection.ClearAllPools();
-
-        var masterAgent = new DatabaseAgent(DatabaseTest.MasterConnectionString);
-        masterAgent.ExecuteBatchFile("Database\\DataDomainObjects_CreateDB.sql", false, DatabaseConfiguration.GetReplacementDictionary());
-        var testDomainAgent = new DatabaseAgent(DatabaseTest.TestDomainConnectionString);
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TestDomain_TearDownDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TableInheritanceTestDomain_TearDownDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TestDomain_SetupDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TableInheritanceTestDomain_SetupDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_Patches.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-
-        _standardMappingDatabaseAgent = new StandardMappingDatabaseAgent(DatabaseTest.TestDomainConnectionString);
-        string sqlFileName = StandardMappingTest.CreateTestDataFileName;
-        _standardMappingDatabaseAgent.ExecuteBatchFile(sqlFileName, true, DatabaseConfiguration.GetReplacementDictionary());
-        string sqlFileName1 = TableInheritanceMappingTest.CreateTestDataFileName;
-        _standardMappingDatabaseAgent.ExecuteBatchFile(sqlFileName1, true, DatabaseConfiguration.GetReplacementDictionary());
-
-        // We don't want the tests to initialize a default mapping; therefore, modify MappingConfiguration.s_fields.Current so that it will 
-        // throw when asked to generate a new MappingConfiguration.
-
-        var throwingMappingConfigurationContainer = new DoubleCheckedLockingContainer<IMappingConfiguration>(
-            () =>
-            {
-              throw new InvalidOperationException(
-                  "This test failed to setup the mapping configuration. Did you forget to derive from StandardMappingTest or to call base.SetUp?");
-            });
-        var fields = PrivateInvoke.GetNonPublicStaticField(typeof(MappingConfiguration), "s_fields");
-        Assertion.IsNotNull(fields);
-        _previousMappingConfigurationContainer = (DoubleCheckedLockingContainer<IMappingConfiguration>)PrivateInvoke.GetPublicField(fields, "Current");
-        PrivateInvoke.SetPublicField(fields, "Current", throwingMappingConfigurationContainer);
-      }
-      catch (Exception ex)
-      {
-        Console.WriteLine("SetUpFixture failed: " + ex);
-        Console.WriteLine();
-        throw;
-      }
+      LogManager.ResetConfiguration();
+      Assert.That(LogManager.GetLogger(typeof(LoggingClientTransactionListener)).IsDebugEnabled, Is.False);
```

Comment:
This code must remain inside the try-catch block to ensure helpful error output. unless nunit3 already provides helpful messages and stacktraces for exceptions during setupfixtures? (please verify)